pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_user-roles-in-pantheon.adoc

```diff
@@ -0,0 +1,20 @@
+[id="user-roles-in-pantheon_{context}"]
+= User roles in {ProductShortName}
+
+In {ProductShortName}, each user is assigned a role that determines the actions the user can perform.
+
+There are two user roles defined in {ProductShortName} - `Author` and `Publisher`. The following permissions are associated with the roles:
+
+`Author`::
+
+* Use Git to import their modules files into {ProductShortName}.
+* Revise existing modules.
+* Preview and stage modules.
+* Search and sort any modules imported into {ProductShortName}.
+* Manage module metadata that is configurable in the user interface.
+ 
```

Comment:
Authors can also create product versions (as opposed to products).

Reply:
@benradey will this be true in Pantheon v2?

Reply:
@stoobie @theashiot Managing products and product versions will be limited to publishers only in Pantheon 2.